mario/tools/tabletransform.py

```diff
@@ -232,3 +232,48 @@ def SUT_to_IOT(instance, method):
     rename_index(matrices["baseline"])
 
     return matrices, indeces, units
+
+
+def ISARD_TO_CHENERY_MOSES(instance,scenario):
+
+    regions = instance.get_index(_MASTER_INDEX['r'])
+    commodities = instance.get_index(_MASTER_INDEX['c'])
+    sN = slice(None)
+
+    U_isard = instance.get_data(['U'], scenarios=[scenario])[scenario][0]
```

Comment:
with the new changes on settings, we need to avoid using the name of matrices directly instead, we need to do like this:
```


from mario.tools.constants import _ENUM

# then instead of using "U", you need to use:
_ENUM.U
# or
_ENUM["U"]

```

This is similar to the _MASTER_INDEX concept

Reply:
done